chapter9/phone-agent/webrtc_app.py

```
@@ -0,0 +1,698 @@
+    except RuntimeError as exc:
+        raise HTTPException(status_code=503, detail=_safe_error(exc)) from exc
+    RUNTIMES[call_id] = runtime
+    pc = RTCPeerConnection()
+    PEERS[call_id] = pc
+    pc.addTrack(runtime.output_track)
+
+    @pc.on("track")
+    def on_track(track: Any) -> None:
+        if track.kind == "audio":
+            record["transport"]["local_audio_track"] = True
+            asyncio.create_task(_consume_microphone(track, runtime))
```

Ruff (0.16.0)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)